recipe/meta.yaml

```diff
@@ -10,16 +10,18 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/jarro2783/cxxopts/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patch:
+  patches:
     - 0001-Fixup-integer-pattern-multiple-definition.patch
 
 build:
   number: 2
 
 requirements:
   build:
-    - toolchain
     - cmake
+    - {{ compiler('c') }}
```

Comment:
Do we need `{{ compiler('c') }}`?

Reply:
I don't know. @isuruf what do you think?

Reply:
No, not necessary.